yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -193,6 +199,11 @@ func buildCommandArguments(dbConfig AssessMigrationDBConfig, exportDirPath strin
 	if dbConfig.Port != "" {
 		args = append(args, "--source-db-port", dbConfig.Port)
 	}
+
+	// Always safe to use --start-clean and --yes to cleanup if there is some state from previous runs in export-dir
+	// since bulk command has separate check to decide beforehand whether the report exists or assessment needs to be performed.
+	args = append(args, "--start-clean", "true",
+		"--yes", "true")
```

Comment:
Why always  defaulting yes to true. Let's propagate the yes arg value from the bulk command to the child assess-migration command instead?

Reply:
done